src/App.js

```diff
@@ -0,0 +1,21 @@
+import { HashRouter, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Layout from './Layout';
+import Home from './components/Home';
+import Details from './components/Details';
+
+function App() {
+  return (
+
+    <HashRouter>
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<Home />} />
+          <Route path="/:Country" index element={<Details />} />
+        </Route>
+      </Routes>
```

Comment:
You successfully created a single page application (SPA) using react-router-dom to serve each page without reloading 😎